packages/cardpay-subgraph/package.json

```diff
@@ -41,7 +41,8 @@
     "sokol-green-status": "ts-node ./lib/sync-status.ts sokol-green",
     "sokol-blue-status": "ts-node ./lib/sync-status.ts sokol-blue",
     "xdai-green-status": "ts-node ./lib/sync-status.ts xdai-green",
-    "xdai-blue-status": "ts-node ./lib/sync-status.ts xdai-blue"
+    "xdai-blue-status": "ts-node ./lib/sync-status.ts xdai-blue",
+    "deploy-local-xdai-fork": "graph deploy local-subgraph --debug-fork QmWawe5A2qv5g9gJE3djFCkvRCqBvt8f6xcJR3JRB64wLu --ipfs http://localhost:5001 --node http://localhost:8020"
```

Comment:
@habdelra I think it may need the full id that you're forking from.

We should script this if so - it'll change every single deployment. The grafting code would be a good first place to check, that calls a meta query for the block number and if not the subgraph I'd I think the full id is available there too.